implementations/rust/ockam/ockam_command/src/environment/mod.rs

```diff
@@ -0,0 +1,13 @@
+use clap::Args;
+
+const ENVTEXT: &str = include_str!("./static/envtext.txt");
+
```

Comment:
You could add a short documentation line for the command like: `Show the Ockam environment variables`